### PR TITLE
Feat: move code for help link in inq-common, add node to guided answers helper

### PR DESCRIPTION
**Pull request by docirl**
https://github.com/SAP/open-ux-tools/issues/2651

- move code for GA link creation from `getHelpForError` to `getHelpLink`
- add node for Service generator to Guided Answers Helper

### CODE REVIEW

**Review by longieirl — DISMISSED** [review]
Changes look good.
Appropiate tests and changeset.

File: packages/inquirer-common/test/unit/error-handler/error-handler.test.ts [hunk 1]
```diff
@@ -307,4 +307,10 @@ describe('Test ErrorHandler', () => {
             t('errors.serverReturnedAnError', { errorDesc: '500 Server Error' })
         );
     });
+
+    test('getHelpForError - getErrorMsgFromType returns undefined', () => {
+        const getErrorMsgFromTypeSpy = jest.spyOn(ErrorHandler, 'getErrorMsgFromType').mockReturnValue(undefined);
+        // @ts-expect-error - Testing undefined return from getErrorMsgFromType
```
Comment (IainSAP) [inline packages/inquirer-common/test/unit/error-handler/error-handler.test.ts:313]: Why is there an error expected here?

**Review by longieirl — APPROVED** [review]
Latest changes look good.

File: packages/inquirer-common/test/unit/error-handler/error-handler.test.ts (hunk 1 above)
Comment (docirl) [reply packages/inquirer-common/test/unit/error-handler/error-handler.test.ts]: Done

**Review by IainSAP — APPROVED** [review]
Thanks @docirl 

- Move of GA related code promptes re-use
- Tests updated

**Pull request merged**